app/jobs/scheduled/clean_up_uploads.rb

```diff
@@ -78,6 +78,7 @@ def execute(args)
         .where("tf.upload_id IS NULL")
         .where("ue.upload_id IS NULL")
         .where("ss.value IS NULL")
+        .where("uploads.access_control_post_id IS NULL")
```

Comment:
I think this `where` should go up, next to the other `where` on the `uploads` table (right before the first `join`).